app/org/sagebionetworks/bridge/StudyConsentBootstrapper.java

```diff
@@ -36,13 +37,17 @@ public StudyConsentBootstrapper(StudyService studyService, StudyConsentService s
             Study study = new DynamoStudy();
             study.setName("Test Study");
             study.setIdentifier("api");
+            study.setSponsorName("Sage Bionetworks");
             study.setMinAgeOfConsent(18);
             study.setResearcherRole("api_researcher");
             study.setConsentNotificationEmail("bridge-testing+consent@sagebridge.org");
+            study.setTechnicalEmail("bridge-testing+technical@sagebridge.org");
+            study.setSupportEmail("support@sagebridge.org");
```

Comment:
Added simple test. Also changed email address (sorry I look in SES and I don't see that address, get confused and set it to the only bridge email address i see there).